Add documentation on validate username adaptive script param (#3805)
wso2/docs-is: en/docs/references/adaptive-authentication-js-api-reference.md

</tbody>
</table>

<a name = "idf-username-validation"></a>
**Username validation for Identifier First handler**

> For the Identifier First handler, by default the username validation will not happen unless the validation is 
> enabled server wide. You can configure the username validation for your applications by specifying 
> `ValidateUsername` property in the `authenticatorParams` object.

Reviewer: ```suggestion
For the Identifier First handler, by default, the username validation will not happen unless the validation is enabled server-wide. You can configure the username validation for your applications by specifying the `ValidateUsername` property in the `authenticatorParams` object.
```

**Example code**

``` java
executeStep(1, {
    authenticatorParams: {
        common: {
            "ValidateUsername": "true"
        }
    }
}, {
    onSuccess: function(context) {
        // Do something on success.
    }
});
```

!!! note

>     Please note this will override the `ValidateUsername` authenticator config that can be configured through the 
>     `deployment.toml` file.

Reviewer: ```suggestion
    Note that these configurations will override the `ValidateUsername` authenticator config in the `deployment.toml` file.
```
